src/webui/src/components/overview/SuccessTable.tsx

```diff
@@ -55,6 +56,15 @@ class SuccessTable extends React.Component<SuccessTableProps, SuccessTableState>
         return items.slice(0).sort((a: T, b: T) => ((isSortedDescending ? a[key] < b[key] : a[key] > b[key]) ? 1 : -1));
     }
 
+    private tooltipStr = (): React.ReactNode => {
```

Comment:
I suppose this can be a variable instead of a function.

Reply:
OK!